sycl/test/basic_tests/queue/queue_parallel_for_generic.cpp

```diff
@@ -0,0 +1,72 @@
+// UNSUPPORTED: cuda
+// CUDA does not support unnamed lambdas.
```

Comment:
Is using "unnamed lambdas" necessary for this test queue API?

Reply:
Done

Reply:
Actually, done from unnamed lambda perspective. But CUDA doesn't work with USM, so added XFAIL to the test

Reply:
I think you can leave UNSUPPORTED instead, but please, update the comment with the justification.

Reply:
Already done. Made it consistent with other bunch of tests. Justification is updated as well